### ISSUE TITLE
Unable to view non-workspace chat

### ISSUE THREAD

**Issue by lossfunc** [thomas-pedersen/cursor-chat-browser#2]
If I use chat in a non-workspace window, can I use this project to browser chat?

**Comment by thomas-pedersen** [comment]
Yes. Even though you don't create a workspace, the chat's are stored in workspace directories.
